### DISCUSSION TITLE
Welcome to docker-freepbx Discussions!

### DISCUSSION THREAD

**Discussion by tiredofit** [tiredofit/docker-freepbx#187] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fclancy** [comment]
I'm looking to run a cloud-based freepbx for my home/office phones. Running a home-based site is problematic with home phones and remote phones and consumer-lever routers/conventional ISPs. Running a docker-based instance on a dedicated cloud server _may_ simplify this for me.

So - my question to begin with is can I run commercial modules (i.e., System Admin, Zulu, Extension Routing, and Endpoint Manager) on this container?

**Reply by tiredofit** [reply]
I believe this would need to be run under a Redhat based container in order to make some of the proprietary modules to work.

**Comment by qxotic** [comment]
I'm using docker V 20.10.12 on a  linux Mint 19 system.  I'm trying to figure out how to get this to run.  I used the stack in portainer to try and load. It failed due to nginx-proxy not being found.  I could not find a wiki and I'm not sure where to go to seek guidance.  
It did download the freepbx, maraidb and db-backup latest files.  Portainer is the only container I have running.  I would only be connecting one phone line.  
Thank you.
